system/Database/Config.php

```diff
@@ -38,8 +39,9 @@ class Config extends BaseConfig
     /**
      * Creates the default
      *
-     * @param array|string $group     The name of the connection group to use, or an array of configuration settings.
-     * @param bool         $getShared Whether to return a shared instance of the connection.
+     * @param array|BaseConnection|string|null $group     The name of the connection group to use,
```

Comment:
Are these newly-supported types? Or a correction of what is actually already acceptable?

Reply:
A correction of what is actually already acceptable.